Yosemite/Yosemite/Payments/CardReaderConnectionStateMachine.swift

```diff
@@ -1,13 +1,68 @@
 import Combine
 import Foundation
-import UIKit
 import Storage
 import SwiftUI
```

Comment:
We can also remove the SwiftUI import from here to assure that we have no UI-related stuff going on, at the moment it still compiles fine when we do 👍